README.md

```diff
@@ -5,64 +5,129 @@ We're excited that you are joining us to dive into the wonders of the [Raspberry
+
+### 3. Set up and configure Resin.io application
+
+To make deploying code on our Pi easy, we're going to use a service called [Resin.io](https://resin.io/).
+To learn more about Resin.io, [click here](https://docs.resin.io/understanding/understanding-code-deployment/).
+
+1. Sign up an account at [Resin.io](https://resin.io/). Ideally, use your GitHub account for authentication.
 
 2. You'll see a prompt to create a new application.
     1. Set Device Type to "Raspberry Pi (v1 or Zero)"
     1. Give your application a name!
     1. Click "Create New Application"
 
-### 4. Set up your Pi!
-TODO: flash pi with resin image
+3. Configure the resinOS image by toggling on "Development" under "Select Edition" and "Wifi + Ethernet" under "Network Connection".
+4. Enter the WiFi SSID and passphrase, and then click "Download".
+
+While the resinOS image is downloading, do the following:
+
+5. Navigate to the ["Preferences" page in the Resin dashboard](https://dashboard.resin.io/preferences/sshkeys) to add your SSH keys. Either select "Import from GitHub" (if you authenticated with GitHub), or enter your public SSH key manually.
+
+You can read more about finding your SSH key or generating a new one [here](https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/).
+You can check to see if you have an existing SSH key by [following the steps here](https://help.github.com/articles/checking-for-existing-ssh-keys/).
+
+6. Set device configuration variable by clicking on "Fleet Configuration". Add the following two device config variables:
+- NAME: `RESIN_HOST_CONFIG_gpu_mem` VALUE: `128`
+- NAME: `RESIN_HOST_CONFIG_start_x` VALUE: `1`
+
+These variables make sure that the camera works on the Pi.
+
+7. Add Resin as a remote. To do so, copy the `git remote add` command in the top right of your Resin dashboard. Run this command in a new tab in your terminal.
+
+### 4. Make a bootable SD card
+
+Once the resinOS image has downloaded and we've completed our configuration steps, we'll use Etcher to flash the image onto our SD card. (You may be asked to grant admin permissions to Etcher.)
+
+1. Open Etcher.
+2. Insert your SD card into your computer.
+3. Using Etcher, click "Select Image" and find the image you downloaded in step 3 above.
+4. Click "Flash!"
+
+Once Etcher finishes creating the bootable SD card, it will eject it for you.
+
+### 5. Set up your Pi
+1. Insert the SD card into your Pi.
+2. Plug your Pi into your computer or a power source using a micro USB cable. (We need power!)
+3. It make take a few minutes, but your Pi should appear on your [Resin dashboard](https://dashboard.resin.io/apps).
+
+4. Enable a public url. Go into `Actions` on your device dashboard and click 'Enable All Public Device URLs'.
+Copy the public url and paste it into `app/static/.env` as the value for `RASPI_URL`.
+```
+RASPI_URL=<your resin machine public url here>
```

Comment:
I would maybe move this to after the "the Rpi button is not working, we're going to fix that" line in the new step 7 but it could remain here since it's easier to do while the url is right there